src/main/kotlin/org/eclipse/tractusx/bpdmcertificatemanagement/entity/CertificateDB.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.eclipse.tractusx.bpdmcertificatemanagement.entity
+
+import jakarta.persistence.*
+import java.time.LocalDateTime
+import java.util.SortedSet
+
+@Entity
+@Table(name = "certificate")
+class CertificateDB(
+
+    @Column(name = "business_partner_number")
+    val businessPartnerNumber: String?,
+
+    @OneToOne(cascade = [CascadeType.ALL], orphanRemoval = true)
+    @JoinColumn(name = "certificate_type_id", unique = true)
+    var type: CertificateTypeDB,
+
+    @Column(name = "registration_number")
+    val registrationNumber: String?,
+
+    @Column(name = "area_of_application")
+    val areaOfApplication: String?,
+
+    @Column(name = "remark")
+    val remark: String?,
+
+    @ElementCollection(fetch = FetchType.EAGER)
+    @CollectionTable(name = "enclosed_sites", joinColumns = [JoinColumn(name = "certificate_id")])
+    val enclosedSites: SortedSet<EnclosedSiteDB>? = sortedSetOf(),
+
+    @Column(name = "valid_from")
+    val validFrom: LocalDateTime?,
```

Comment:
Saving this as Instant in the database makes it explicit that it should be saved in UTC. Alternatively you could go with a time zoned type so the database would also store the time zone and give the original time zone back. Could be an advantage to avoid multiple users putting in dates in various time zones.

Reply:
Done